Dockerfile.rpi

```diff
@@ -4,7 +4,16 @@ COPY 01_nodoc /etc/dpkg/dpkg.cfg.d/
 
 # Supervisor apt dependencies
 RUN apt-get -q update \
-	&& apt-get install -qqy socat supervisor --no-install-recommends \
+	&& apt-get install -qqy \
+		btrfs-tools \
+		ca-certificates \
+		curl \
+		docker.io \
```

Comment:
@petrosagg do we have any way to ensure this is also 1.6.2? Maybe do docker.io=$DOCKER_APT_VERSION and generate DOCKER_APT_VERSION from DOCKER_VERSION?